Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocList.css

```diff
@@ -361,6 +361,13 @@ th.bocListTitleCell > a[href]:active
   background-color: #C5E0FC;
 }
 
+th.bocListTitleCell > a > img,
+th.bocListTitleCell > span > img
```

Comment:
Oh no, I found a css bug in NovaGray: we need to duplicate the title-rules for TD elements. please create a ticket, thanks! This was changed when we introduced column title referneces for screenreaders.

Reply:
I don't think I have enough information to create a bug ticket from your comment.